src/objects.js

```diff
@@ -1,41 +1,67 @@
 const createPerson = (name, age) => {
-  // your code here
+  return {
+    'name': name,
+    'age': age 
+  };
 };
 
 const getName = object => {
-  // your code here
+  return object.name;
 };
 
 const getProperty = (property, object) => {
-  // your code here
+  return object[property];
 };
 
 const hasProperty = (property, object) => {
-  // your code here
+  if (object[property]) {
+    return true;
+  }
+  return false;
 };
 
 const isOver65 = person => {
-  // your code here
+  if (person.age > 65) {
+    return true;
+  }
+  return false;
 };
 
 const getAges = people => {
-  // your code here
+  return people.map(index => index.age);
 };
 
 const findByName = (name, people) => {
-  // your code here
+  return people.filter(index => index.name === name)[0];
 };
 
 const findHondas = cars => {
-  // your code here
+  let hondaCars = [];
+  for (const i in cars) {
+    if (cars[i].manufacturer === 'Honda') {
+      hondaCars.push(cars[i]);
+    }
+  }
+  return hondaCars;
 };
 
 const averageAge = people => {
-  // your code here
+  let avrAge = 0;
+  for (const i in people) {
+    avrAge += Math.round(people[i].age / people.length);
+  }
+  return avrAge;
+  // Could try using .reduce() as another method of getting one number from mulitple //
```

Comment:
Reduce is definitely the right way to go!
```
  const total = people.reduce((previous, current) => {
    return previous + current.age;
  }, 0);

  return total / people.length;
```

This could be simplified a bit further to avoid creating the variable:
```suggestion
return (
    people.reduce((previous, current) => {
      return previous + current.age;
    }, 0) / people.length
  );
```